spec/std/crypto/bcrypt/password_spec.cr

```diff
@@ -46,7 +52,7 @@ describe "Crypto::Bcrypt::Password" do
 
   describe "verify" do
     password = Crypto::Bcrypt::Password.create("secret", 4)
-    password2 = Crypto::Bcrypt::Password.new("$2b$04$ZsHrsVlj.dsmn74Az1rjmeE/21nYRC0vB5LPjG7ySBfi6lRaO/P22")
+    password2 = Crypto::Bcrypt::Password.new("$2$04$ZsHrsVlj.dsmn74Az1rjmeE/21nYRC0vB5LPjG7ySBfi6lRaO/P22")
```

Comment:
We should definitely have a check that `2b` is supported.

Reply:
If I understand correctly, yes, this will be caught as expected by the code above, which raises if it's not in the list of supported versions.

Reply:
We're missing a positive test for a `$2b$` password hash.